controllers/commentController.js

```diff
@@ -142,13 +137,182 @@ export default class CommentController {
         ]
       });
       if (!comment) {
-        res
-          .status(BAD_REQUEST)
+        return res
+          .status(NOT_FOUND)
           .json({ message: "There is no comment with that id" });
       }
       res.status(OK).json({ comment });
     } catch (error) {
-      res.status(INTERNAL_SERVER_ERROR).json({ message: "Sorry, this is not working properly. We now know about this mistake and are working to fix it" });
+      res.status(INTERNAL_SERVER_ERROR).json({
+        message:
+          "Sorry, this is not working properly. We now know about this mistake and are working to fix it"
+      });
+    }
+  }
+
+  /**
+   * @description Allows users to fetch a particular comment of an article.
+   * @param  {Object} req - The request object.
+   * @param  {Object} res - The response object.
+   * @returns {Object} - It returns the request response object.
+   */
+  static async fetch(req, res) {
+    try {
+      const { id: commentId } = req.params;
+      const comment = await Comment.findOne({
+        where: { id: commentId },
+        include: [
```

Comment:
@YvesIraguha You only need likes' count because likes might become too many to be fetched in a single request